Modules/AirCastingStyling/Sources/AirCastingStyling/UnFollowButtonStyle.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  File.swift
+//  
+//
+//  Created by Lunar on 19/07/2021.
+//
```

Comment:
those headers don't seem consistent with the rest of the codebase

Reply:
What do you mean? This header is the same as other headers in the AirCastingStyling module.